src/pages/home/report/ReportActionItemSingle.js

```diff
@@ -51,13 +52,14 @@ const showUserDetails = (email) => {
 };
 
 const ReportActionItemSingle = (props) => {
+    const actorEmail = props.action.actorEmail.replace(CONST.REGEX.MERGED_ACCOUNT_PREFIX, "");
```

Comment:
the merged accounts will have `MERGED_<number>@` prefix on the `actorEmail` so lets remove that so we can load the details correctly